test_sensu.yml

```diff
@@ -4,5 +4,8 @@
   vars:
       firewall_manage_rules: true
   roles:
+    - redis
+    - rabbitmq/sensu
```

Comment:
I think this is the wrong way to get the rabbitmq user created for sensu.  I think we should use the structure I proposed in #41, and then we can use ansible host delegation to have this be a part of the sensu role but execute on the servers running rabbitmq.  E.g.

```
- name: Create sensu user in rabbitmq
  delegate_to: rabbitmq_hosts
  rabbitmq_user:
    user: '{{ sensu_rabbitmq_user }}'
    ...
```